lib/practice_exercises.rb

```diff
@@ -1,13 +1,63 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: n log(n)
+# Space Complexity: n
 def is_palindrome(string)
```

Comment:
👍 You are not doing sorting or anything other than loops which go through the string so this is O(n) time complexity.